doc/users/colors.rst

```diff
@@ -17,6 +17,7 @@ it can be provided as:
 * a name from the `xkcd color survey <https://xkcd.com/color/rgb/>`__
   prefixed with ``'xkcd:'`` (e.g., ``'xkcd:sky blue'``)
 * one of ``{'C0', 'C1', 'C2', 'C3', 'C4', 'C5', 'C6', 'C7', 'C8', 'C9'}``
+* one of ``{'blue', 'orange', 'green', 'red', 'purple', 'brown', 'pink', 'gray', 'olive', 'cyan'}``
```

Comment:
Thanks for thinking about documenting this change! That's a really important part of making new features accessible to users.

I think you should add the "vega" prefix to be more specific.